src/main/java/pages/HomePage.java

```diff
@@ -21,4 +43,65 @@ public CatalogPage clickOnCatalogsPageButton() {
         logger.debug("Clicked successfully on the catalog page button");
         return new CatalogPage();
     }
+
+    @Step("Click on the services page button")
+    public ServicesPage clickOnTheServicesPageButton() {
+        servicesButton.click();
+        logger.debug("Clicked successfully on the services page button");
+        return new ServicesPage();
+    }
+
+    @Step("Click on the logotype")
+    public HomePage clickOnTheLogotype() {
+        logo.click();
+        logger.debug("Clicked successfully on the logo");
+        return new HomePage();
+    }
+
+    @Step("Get page url")
+    public String getPageUrl() {
+        logger.debug("Successfully retrieved current URL");
+        return driver.getCurrentUrl();
+    }
+
+    @Step("Input text in the search field")
+    public HomePage textInput() {
+        searchField.sendKeys("Дом в Минске");
```

Comment:
DONT HARDCODE test data inside methods!!!!
To inserst different string you are going to create a separate method with the same functionality?